```
1/18/2022 11:04:52 AM
Status: Completed
##[error] Installer https://github.com/RickStrahl/MarkdownMonsterReleases/blob/master/v2/v2.3/MarkdownMonsterSetup-2.3.1.exe
##[error] Result: Failed
##vso[task.logissue type=error] Specified hash doesn't match.
Actual hash:   D178307AF73E836C2835B63C54976DE2B95095892F6B65ABC1188700DBFAEDE1
Expected hash: f1591633acdd168431dd773965b79ed830c414ec900a86244356534ed092442c
```

Probably caching issues, seems like they updated the hash a few times

--- a/manifests/w/WestWind/MarkdownMonster/2.3.1.0/WestWind.MarkdownMonster.installer.yaml
+++ b/manifests/w/WestWind/MarkdownMonster/2.3.1.0/WestWind.MarkdownMonster.installer.yaml
@@ -15,6 +15,6 @@ Dependencies:
 Installers:
 - Architecture: x64
   InstallerUrl: https://github.com/RickStrahl/MarkdownMonsterReleases/blob/master/v2/v2.3/MarkdownMonsterSetup-2.3.1.exe
-  InstallerSha256: F1591633ACDD168431DD773965B79ED830C414EC900A86244356534ED092442C
+  InstallerSha256: 2E2823FA4558115465E031F86EC1334E8B09F380918B64D91FB162844129E804
 ManifestType: installer
 ManifestVersion: 1.1.0

--- a/manifests/w/WestWind/MarkdownMonster/2.3.1.0/WestWind.MarkdownMonster.installer.yaml
+++ b/manifests/w/WestWind/MarkdownMonster/2.3.1.0/WestWind.MarkdownMonster.installer.yaml
@@ -14,7 +14,7 @@ Dependencies:
   - PackageIdentifier: Microsoft.EdgeWebView2Runtime
 Installers:
 - Architecture: x64
-  InstallerUrl: https://github.com/RickStrahl/MarkdownMonsterReleases/blob/master/v2/v2.3/MarkdownMonsterSetup-2.3.1.exe
-  InstallerSha256: 2E2823FA4558115465E031F86EC1334E8B09F380918B64D91FB162844129E804
+  InstallerUrl: https://github.com/RickStrahl/MarkdownMonsterReleases/raw/master/v2/v2.3/MarkdownMonsterSetup-2.3.1.exe
+  InstallerSha256: DA2ADD30DC86E83F26AB7BE041AB810117ED86CD86415419CB63177737E09208
 ManifestType: installer
 ManifestVersion: 1.1.0

